static/js/biomedical/protein/data_processing_utils.ts

```diff
@@ -528,7 +544,9 @@ export function getChemicalGeneAssoc(data: GraphNodes): ProteinNumData[] {
     for (const m in chemAssoc) {
       result.push({ name: m, value: chemAssoc[m] });
     }
-
+    if (check_for_zeroes(result)) {
```

Comment:
nit: don't really need the helper function because could just check that there is at least one value that is not zero like:
```
const hasNonZero = result.findIndex((obj) => obj.value !== 0) >= 0;
if (!hasNonZero) {
...
}
```

Reply:
great suggestion! done!